Add supervised learning info on EPPI reviewer (#25)
Rensvandeschoot/software-overview-machine-learning-for-screening-text: README.md

## Supervised Learning

|                  Software                   | Feature Extr. |          Classifiers           |          Balancing          | Query Stra. |
|:-------------------------------------------:|:-------------:|:------------------------------:|:---------------------------:|:-----------:|
| [EPPI-Reviewer](#eppi-reviewer)<sup>1</sup> |    TF-IDF     | SVM:grey_question:<sup>2</sup> | :grey_question:<sup>1</sup> |  R, C, Cl   |

> <sup>1</sup> Besides building your own classifier, EPPI-Reviewer offers the option to choose from pre-trained models to find a specific type of literature, for e.g., RCTs.

Reviewer: ```suggestion
<sup>1</sup> EPPI-Reviewer offers the option to choose from, or use custom, pre-trained models to find a specific type of literature, e.g., for RCTs.
```
